server/log_operation_manager.go

```diff
@@ -519,3 +465,86 @@ loop:
 	l.runnerWG.Wait()
 	glog.Infof("wait for termination of election runners...done")
 }
+
+// logOperationExecutor runs the specified LogOperation on the submitted logs
+// in a set of parallel workers.
+type logOperationExecutor struct {
+	op   LogOperation
+	info *LogOperationInfo
+
+	// jobs holds logIDs to run log operation on.
+	// TODO(pavelkalinnikov): Use mastership context for each job to make them
+	// auto-cancelable when mastership is lost.
+	// TODO(pavelkalinnikov): Report job completion status back.
+	jobs chan int64
+}
+
+func newExecutor(op LogOperation, info *LogOperationInfo, jobs int) *logOperationExecutor {
```

Comment:
(nit) could make this a smidgeon shorter with:
```go
if jobs < 0 {
  jobs = 0
}
return &logOperationExecutor{op: op, info: info, jobs: make(chan int64, jobs)}
```

Reply:
Done.